cocos/core/assets/texture-cube.ts

```diff
@@ -59,6 +78,24 @@ enum FaceIndex {
     front = 4,
     back = 5,
 }
+export enum MipmapBakeMode {
```

Comment:
MipmapMode {
    NONE = 0,
    AUTO = 1,
    BAKED_ATLAS = 2,
}

would be better, texture cube doesn't need to know where the baked mipmap comes from, it only cares whether need to generate mipmaps or not